src/app/components/observation-goals/observation-goals.component.html

```diff
@@ -0,0 +1,85 @@
+                        <span i18n>Name</span>
+                        <em [ngClass]="getSortClass('name')" class="ml-2"></em>
+                    </th>
+                    <th class="pointer w-25 hover-shadow" scope="col" (click)="setSort('description')">
+                        <span i18n>Description</span>
+                        <em [ngClass]="getSortClass('description')" class="ml-2"></em>
+                    </th>
+                    <th class="pointer w-20 hover-shadow" scope="col" (click)="setSort('legalGround')">
+                        <span i18n>Legal Ground</span>
+                        <em [ngClass]="getSortClass('legalGround')" class="ml-2"></em>
+                    </th>
+                    <th class="pointer w-20 hover-shadow" scope="col" (click)="setSort('legalGroundLink')">
+                        <span i18n>Legal Ground Link</span>
+                        <em [ngClass]="getSortClass('legalGroundLink')" class="ml-2"></em>
+                    </th>
+                    <th class="w-10" scope="col">
+                        <span i18n>Actions</span>
+                    </th>
+                </tr>
+                </thead>
+                <tbody>
+                <tr *ngFor="let observationGoal of observationGoals; let i = index" class="pointer">
+                    <th scope="row">{{ (pageSize * pageIndex) + i + 1 }}</th>
+                    <td>{{ observationGoal.name }}</td>
+                    <td>{{ observationGoal.description }}</td>
+                    <td>{{ observationGoal.legalGround }}</td>
+                    <td>{{ observationGoal.legalGroundLink }}</td>
+                    <td class="px-1 pt-1 pb-0 m-0">
+                        <button type="button" class="btn btn-primary btn-sm p-1 mt-1 mb-1 mx-1" (click)="editObservationGoal(observationGoal._id)">
+                            <span>
+                                <i class="fas fa-pencil-alt m-1"></i>
+                            </span>
+                        </button>
+                        <button type="button" class="btn btn-danger btn-sm p-1 mt-1 mb-1 mx-0" (click)="removeObservationGoal(observationGoal._id)">
+                            <span>
+                                <i class="fas fa-trash-alt m-1"></i>
+                            </span>
+                        </button>
+                    </td>
+                </tr>
+                </tbody>
+            </table>
+            <nav class="d-flex fixed-bottom justify-content-center mb-4">
+                <ul class="pagination mb-0">
+                    <li class="pointer" (click)="getPreviousPage()">
+                        <span><em class="fa fa-chevron-left"></em></span>
+                    </li>
+                    <li class="pointer active">
+                        <span class="content">{{ pageIndex + 1}}</span>
+                    </li>
+                    <li class="pointer" (click)="getNextPage()">
```

Comment:
Mini dingetje, je kunt naar volgende pagina's blijven gaan, terwijl je alle entries al hebt gehad. Wil je dat afvangen in `getNextPage()`? Simpele check op `len(huidige_lijst) < pageSize`